### ISSUE TITLE
Add support for SegaCD/MegaCD .chd file scraping

### ISSUE THREAD

**Issue by LemaruX** [sselph/scraper#215]
Is it possible to implement scraping of SegaCD/MegaCD images which are in .chd format. Also to retain the "Disk1" or "Disk2" naming convention for multi disk games?

**Comment by clownsec** [comment]
I second this, this is the only piece of the puzzle missing for me.  I may be one of the few who LOVED the segacd system, and I have all roms in chd but am going to have to create a huge amount of work creating the gamelist.xml to display video preview.

**Comment by einsteinx2** [comment]
Same issue with Dreamcast, which isn't helped by the fact that Reicast and Demul support CHDv4 while Redream supports both CHDv4 and the much better CHDv5.

Though I think in general, due to the fact that this scraper works on hashing and people are manually converting their images to CHD, I'm not sure if CHD compression is deterministic or not. E.g. does compressing the same disc image create the exact same CHD file or will the hashes be different?

**Comment by LemaruX** [comment]
Good point as the hash will always be different depending on what was used to rip/compress it I imagine? Could another identifier not be used, maybe even filename as a last resort?
